### ISSUE TITLE
ShaderCompiler should support get${Tex}AtOutCoords / broadcasting for packed textures

### ISSUE THREAD

**Issue by annxingyuan** [tensorflow/tfjs#803]
Currently, packed kernels such as batch normalization manage broadcasting internally. This should be abstracted by the shaderCompiler.
